client/reader/following-edit/style.scss

```diff
@@ -109,8 +108,20 @@
 }
 
 .following-edit__search {
-	color: $blue-wordpress;
-	cursor: pointer;
+	margin-right: 0;
+	padding: 0;
+
+	.gridicons-search {
+		color: $blue-wordpress;
+	}
+
+	&.button.is-borderless .gridicons-search {
+		top: 4px;
+	}
+
+	&:focus .gridicons-search {
+		color: $blue-light;
```

Comment:
How about a border on focus instead, to be consistent with the other buttons?

Something like...

`border-color: #00aadc;
box-shadow: 0 0 0 2px #78dcfa;
padding-top: 8px;
`